packages/actor-optimize-query-operation-filter-pushdown/lib/ActorOptimizeQueryOperationFilterPushdown.ts

```diff
@@ -161,6 +174,25 @@ export class ActorOptimizeQueryOperationFilterPushdown extends ActorOptimizeQuer
       return true;
     }
 
+    // Don't push down extension functions comunica support, but an endpoint doesn't
```

Comment:
I think there are some issues with this logic here:

1. If there's just a single source that supports the extension function, we should still push down, but then only call the extension functions client-side for those endpoints having no support.
2. In order to do 1., we may need some changes in `filterPushdown` (but not 100% certain). I'd start with adding some tests for expected behaviour, and implementing that in a test-driven approach.